tests/lts_compatibility.py

```diff
@@ -501,10 +503,25 @@ def run_ledger_compatibility_since_first(args, local_branch, use_snapshot):
                         version,
                     )
 
+                # We accept ledger chunk file differences during upgrades
+                # from 2.x-rc6 to 2.x-rc7 and from 1.x to 2.x-rc7
```

Comment:
I think we should explain why too, and perhaps link to #3613 so that we can clean this up once 2.0 is released.